app.py

```diff
@@ -6,25 +6,25 @@
 
```

Comment:
`app.config.from_object('flask_config.Config')` can be removed as the `SECRET_KEY` is no longer required (we're not using the flask session cookie anymore).